src/transaction/transaction_group_complete_manager.cpp

```diff
@@ -39,55 +46,167 @@ namespace cubtx
 
     on_register_transaction ();
 
+    er_log_debug (ARG_FILE_LINE, "group_complete_manager::register_transaction: (tran_index=%d, MVCCID=%llu, " \
+		  "tran_state=%d, current group size = %d)\n", tran_index, (long long int) mvccid, (int) state,
+		  m_current_group.get_container ().size ());
+
     return m_current_group_id;
   }
 
   //
-  // wait_for_complete_mvcc waits for MVCC complete event on specified group_id.
+  // complete_mvcc complete transactions MVCC.
```

Comment:
remove comment  or change it to shortly describe what the method actually does : is called by the transaction thread and blocks until group is marked as MVCC completed by the GCM thread

Reply:
In SA, there is no wait. The comment adds transaction word.